### PR TITLE
Update point 15 and add point 17 on API help page #374

### PR DESCRIPTION
**Pull request by kencho51**
# Pull request for issue: #374 

This is a pull request for the following changes:
Implement the updates requested by @only1chunts 
Update title and link for point 15
Add Point 17

### CODE REVIEW

File: files/html/help.html [hunk 1]
```diff
@@ -490,10 +490,12 @@ <h3><a name="0.1_Examples">Examples</a></h3><br>
...
 <div>14. refine search to project names</div>
 <div><a href="/api/search?project=Genome%2010K&amp;result=sample" target="_blank">http://gigadb.gigasciencejournal.com/api/search?<wbr>project=Genome%2010K&amp;result=<wbr>sample</a></div><br>
-<div>15. list all dataset doi</div>
-<div><a href="/api/list" target="_blank">http://gigadb.gigasciencejournal.com/api/list</a></div><br>
+<div>15. list all pulished dataset DOIs (listed in publication date order)</div>
+<div><a href="/api/list" target="_blank">http://gigadb.org/api/list</a></div><br>
 <div>16. dump the database</div>
 <div><a href="/api/dump" target="_blank">http://gigadb.gigasciencejournal.com/api/dump</a></div><br>
+<div>17. list all dataset DOI's published in a date range (results ordered by publication date)</div>
+<div><a href="/api/list" target="_blank">http://gigadb.org/api/list?start_date=2018-01-01&end_date=2018-01-30</a></div><br>
```
Comment (only1chunts) [inline files/html/help.html:498]: I think href here is incorrect, it doesn't match the URL text

**Comment by kencho51** [comment]
@only1chunts Your new numbered points (15 and 17) starts with capital letter, which is inconsistent with the numbered points already present, I committed them with  small letters to keep the consistency.

**Comment by only1chunts** [comment]
@kencho51, its great that you've taken the time to think about the consistency of the page and used your judgement, much appreciated, thank you.

**Comment by pli888** [comment]
@kencho51 When I look in the `Files changed` tab, it shows only that a file `files/html/.help.html.swp` has been added. There are no changes to `help.php` or `help.html.dist` in this pull request.

**Pull request merged**